src/Entity/ProcurationProxy.php

```diff
@@ -737,63 +744,47 @@ public function isFrenchRequestAvailable(): bool
         return $this->frenchRequestAvailable;
     }
 
-    private function setFrenchRequestAvailable(bool $frenchRequestAvailable): void
-    {
-        $this->frenchRequestAvailable = $frenchRequestAvailable;
-    }
-
     public function isForeignRequestAvailable(): bool
     {
         return $this->foreignRequestAvailable;
     }
 
-    private function setForeignRequestAvailable(bool $foreignRequestAvailable): void
+    public function process(ProcurationRequest $request): void
     {
-        $this->foreignRequestAvailable = $foreignRequestAvailable;
+        $this->addFoundRequest($request);
+
```

Comment:
Useless whitespace

Reply:
`useless` is a big word :p

i put a whitespace here for readability. 2 separate steps are done in this method.

Reply:
but well. ok. you're right, as always Ô @tony-tran